.circleci/config/jobs/deploy_to_dev.yml

```diff
@@ -14,10 +14,28 @@ steps:
         LATEST_TAG_WITH_V=`git describe --abbrev=0 --tags ${CIRCLE_SHA1}` &&
         LATEST_TAG=${LATEST_TAG_WITH_V:1}-approved &&
         ./scripts/slack/notify_deploy.py $LATEST_TAG dev &&
-        curl -s --fail --show-error -i -H "Accept: application/json" -H "Content-Type: application/json" \
-            -X POST -d "{\"docker_sha\":\"${LATEST_TAG}\", \
-                          \"commit_hash\":\"${CIRCLE_SHA1}\"}" \
-            https://my.dev.snyk.io/${DEV_DEPLOY_TOKEN}
+        git clone https://$GH_TOKEN@github.com/snyk/$KUBERNETES_MONITOR_DEPLOYER_REPO.git
+        mkdir -p $KUBERNETES_MONITOR_DEPLOYER_REPO/helm/values/
+        cat >$KUBERNETES_MONITOR_DEPLOYER_REPO/helm/values/pre-prod-mt.yaml <<EOF
+        clusterName: "Development cluster"
+        integrationApi: https://kubernetes-upstream.dev.snyk.io
+        log_level: "DEBUG"
+        policyOrgs: {$POLICY_ORG_PRE_PROD}
```

Comment:
Nit: `${`

Reply:
Actually wait, this is Helm array syntax?

Reply:
It's an object that looks like `{uuid}`, so I believe it's correct here